Automattic/jetpack
# Social: Hide settings button on Simple sites on the auto-convert notice

This makes it so on WPCOM sites the auto-conversion notice should not have the `Change settings` button, as they cannot turn the feature off as of now.

## Proposed changes:

* Added a prop to be able to hide the settings button 58609dd3721c8de62e593ac5698d7b89dca2a22d
* Added the check to hide the settings on Simple sites 0ac3ec30e937a1564e4f53db0b6c542965eaed89
* Added some tests for the component to increase our coverage 💪 5487c45bec5530f9bf81c483d9015f57d71e512b
* Since we hid the notice before I unhid it here 512744355d82b4c2d1adb1f348ce3756ab6b4d6b

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=46760258
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* Until D131678-code is merged, you need to follow the Test plan on that patch and check out that patch on your sandbox.
  * On your `wpsh` sandbox check out the branch `arc patch D131678`
  * With Jetpack downloader download this PR onto the checked out branch with `bin/jetpack-downloader test jetpack add/social-conversion-notice-wpcom`.
  * When prompted, say not to for checking out trunk, then say yes in the next question to continue.
  
Now you should have both changes applied.

* Make sure you have an Instagram Business connection
* Start editing a new post and add an invalid but convertible media as a featured image - for example a 10Mb size jpg image
* You should see that the image will be converted, but the `Change settings` button should not appear.
* Clicking `Got it` and refreshing the page should not bring back the notice as it's dismissed.
